composer.json

```diff
@@ -5,11 +5,14 @@
     "require": {
         "sparkphp/spark": "dev-master",
         "lichtner/fluentpdo": "dev-master",
-        "robmorgan/phinx": "dev-master"
+        "robmorgan/phinx": "dev-master",
+        "nesbot/carbon": "dev-master",
+        "firebase/php-jwt": "dev-master"
```

Comment:
In the future, we might want to add [Underscore](https://github.com/Im0rtality/Underscore) too. I worked on it heavily during my two weeks off and it is shaping up nicely.